### PR TITLE
IACO-605: Document registration

### PR DESCRIPTION
**Pull request by grozen**
As with all documentation pull requests, you may want to look at this [rendered](https://github.com/klarna/klarna-on-demand-android/blob/feature/IACO-605-document-registration/doc/integration.md).

### CODE REVIEW

File: doc/integration.md [hunk 1]
```diff
@@ -1 +1,110 @@
-Nothing here just yet!
+#Integration Guide
+This guide includes all information necessary to receive payments from a user of your application through Klarna. In this guide, you will see how to allow the user to register his device with Klarna, change payment preferences and perform purchases.
+
```
Comment (yuval-netanel) [inline doc/integration.md:3]: I think that we are missing table of contents with links (also in the IOS):
- Set API key
- Registration
- Purchase
- Preference

File: doc/integration.md [hunk 2]
```diff
@@ -1 +1,110 @@
...
+**Note:** It is important to point out that the registration activity will not function properly without network access, and that it does not currently support a landscape orientation.
+
+###Showing the view
+For the sake of this example, assume we have a button that launches the registration activity (we will cover a better approach [later](#when_to_show_registration)).
+
+First off, import the registration activity:
+
+```java
+import com.klarna.ondemand.RegistrationActivity;
```
Comment (yuval-netanel) [inline doc/integration.md]: I think you will need to import REGISTRATION_REQUEST_CODE as well

**Comment by yuval-netanel** [comment]
:cool:

**Comment by tomerlevi444** [comment]
Approved!
Please make sure to merge with master and build, before merging.

**Pull request merged**